### PR TITLE
Add a timeline section

### PR DESCRIPTION
**Pull request by roxelo**
image
image

### CODE REVIEW

File: content/introduction.html [hunk 1]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+        <li>SEXXI start at the University of Waterloo</li>
+        <li>First shortened Orientation Week</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Winter 2017</h3>
+    <section>
+      <ul>
+        <li>r/uwaterloo wins the meme war against r/UofT thanks to a <a href="https://www.reddit.com/r/uwaterloo/comments/5ibr95/fuck_it_if_this_post_receives_2000_upvotes_i_will/">goose tatoo</a></li>
```
Comment (jenniferwills) [inline content/introduction.html]: tattoo
```suggestion
        <li>r/uwaterloo wins the meme war against r/UofT thanks to a <a href="https://www.reddit.com/r/uwaterloo/comments/5ibr95/fuck_it_if_this_post_receives_2000_upvotes_i_will/">goose tattoo</a></li>
```

File: content/introduction.html [hunk 2]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+        <li>SEXXI use (ie: beta test) Waterlooworks</li>
```
Comment (jenniferwills) [inline content/introduction.html]:
```suggestion
        <li>SEXXI use (ie: beta test) WaterlooWorks</li>
```

**Review by jenniferwills — APPROVED** [review]
other than spelling lgtm

File: content/introduction.html [hunk 3]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
 future classes, as we did, in order to take future profiles to new heights.
 </p>
 
+<div class="vbx6"></div>
+
+<h1> Timeline </h1>
+<p class="full">
+  Within our time at Waterloo, the Software Class of 2021 (a.k.a SEXXI) experienced many firsts and witnessed many note
```
Comment (jaxonL) [inline content/introduction.html]:
```suggestion
  During our time at Waterloo, the Software Class of 2021 (a.k.a SEXXI) experienced many firsts and witnessed various```

File: content/introduction.html [hunk 4]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+  worhy events that we have noted for you below.
```
Comment (jaxonL) [inline content/introduction.html]:
```suggestion
  noteworthy events, which we list for you below.
```

File: content/introduction.html (hunk 2 above)
Comment (jaxonL) [reply content/introduction.html]: and "i.e." instead of "ie" as well?

File: content/introduction.html [hunk 5]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+        <li>SEXXI use (ie: beta test) Waterlooworks</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Fall 2017</h3>
+    <section>
+      <ul>
+        <li>New Residence Building (NRB) opens on campus</li>
+        <li>NRB is renamed Claudette Milla Hall (CMH) 2 months later</li>
```
Comment (jaxonL) [inline content/introduction.html]:
```suggestion
        <li>NRB is renamed Claudette Millar Hall (CMH) 2 months later</li>
```

File: content/introduction.html [hunk 6]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+      <ul>
+        <li>Lime Scooter Pilot</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Winter 2019</h3>
+    <section>
+      <ul>
+        <li>First UWaterloo Snow Day</li>
```
Comment (jaxonL) [inline content/introduction.html:148]:
```suggestion
        <li>The Bombshelter Pub (a.k.a "The Bomber"), UWaterloo's on-campus pub, closes</li>
        <li>First UWaterloo Snow Day</li>
        <li>Feds (Federation of Students) starts rebranding movement, eventually settles on WUSA (Waterloo Undergraduate Student Association)</li>
```
additional event suggestions

File: content/introduction.html [hunk 7]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Summer 2019</h3>
+    <section>
+      <ul>
+        <li>ION opens</li>
```
Comment (jaxonL) [inline content/introduction.html]:
```suggestion
        <li>ION, Waterloo's light rail transit system, opens</li>
```

idk how necessary it is to explain this

File: content/introduction.html [hunk 8]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+      <ul>
+        <li>Initial predicted completion date of SLC PAC Extension</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Winter 2020</h3>
+    <section>
+      <ul>
+        <li>Global Pandemic</li>
```
Comment (jaxonL) [inline content/introduction.html]:
```suggestion
        <li>Global Pandemic (Covid-19)</li>
```

(i mean, i don't think we'll have another one soon, but we can mention the virus by name maybe for the future generation who don't know of the pandemic?

File: content/introduction.html [hunk 9]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+      <ul>
+        <li>SLC PAC Extension start</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Fall 2018</h3>
+    <section>
+      <ul>
+        <li>Lime Scooter Pilot</li>
```
Comment (jaxonL) [inline content/introduction.html:137]:
```suggestion
        <li>Lime Scooter Pilot</li>
        <li>Engineering 7's construction is complete, and the building is officially open to students</li>
```

File: content/introduction.html [hunk 10]
```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
...
+        <li>New Residence Building (NRB) opens on campus</li>
+        <li>NRB is renamed Claudette Milla Hall (CMH) 2 months later</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Summer 2017</h3>
+    <section>
+      <ul>
+        <li>SLC PAC Extension start</li>
```
Comment (jaxonL) [inline content/introduction.html]:
```suggestion
        <li>SLC PAC Extension starts</li>
```

**Review by jaxonL — COMMENTED** [review]
some rewording nits and 3 additional event suggestions (e7 opening, feds -> wusa rebranding, bomber closing)

File: content/introduction.html [hunk 11]
```diff
@@ -69,6 +69,114 @@ <h1> Preface </h1>
 future classes, as we did, in order to take future profiles to new heights.
 </p>
 
+<div class="vbx6"></div>
+
+<h1> Timeline </h1>
+<p class="full">
+  During our time at Waterloo, the Software Class of 2021 (a.k.a SEXXI) experienced many firsts and witnessed various
+  noteworhy events, which we list for you below.
```
Comment (jaxonL) [inline content/introduction.html]:
```suggestion
  noteworthy events, which we list for you below.
```

**Review by jaxonL — APPROVED** [review]
LGTM 💯 (apart from small typo)

**Comment by roxelo** [comment]
While I was updating the timeline I also noticed that some of the interactive graphs did not include the generic interactive message ```<b> Interactive: </b> Click on a provided filter to view the plot for that filter.``` So I also added that in this PR

**Pull request merged**